test/worlds/air_pressure.sdf

```diff
@@ -5,6 +5,10 @@
       filename="ignition-gazebo-air-pressure-system"
       name="ignition::gazebo::systems::AirPressure">
     </plugin>
+    <plugin
+      filename="libignition-gazebo-scene-broadcaster-system.so"
+      name="ignition::gazebo::systems::SceneBroadcaster">
+    </plugin>
```

Comment:
Is this needed for the test?

Reply:
I believe so, the test didn't work without it. I can try again

Reply:
Yeah, it's needed. I just removed all the plugins, which has the same effect but looks cleaner. f32131b